Civi/Api4/Query/SqlFunctionISNOTNULL.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Query;
+
+/**
+ * Sql function
+ */
+class SqlFunctionISNOTNULL extends SqlFunction {
+
+  protected static $category = self::CATEGORY_COMPARISON;
+
+  protected static $dataType = 'Boolean';
+
+  protected static function params(): array {
+    return [
+      [
+        'optional' => FALSE,
+      ],
+    ];
+  }
+
+  /**
+   * @return string
+   */
+  public static function getTitle(): string {
+    return ts('Is not null');
+  }
+
+  /**
+   * @return string
+   */
+  public static function getDescription(): string {
+    return ts('FALSE if the value is NULL, otherwise TRUE.');
+  }
+
+  /**
+   * Get the name of this sql function.
+   * @return string
+   */
+  public static function getName(): string {
+    return 'ISNULL';
+  }
```

Comment:
This function is returning the wrong name. But also the function is unnecessary. It can be removed as the parent provides this.

Reply:
Ah I thought the comment of the function indicated that I should return the mysql function. But it is the system name of the function class. Will resolve this.